clifford/test/test_clifford.py

```diff
@@ -741,6 +741,24 @@ def test_right_multiplication_matrix(self, algebra, rng):  # noqa: F811
             res2 = layout.MultiVector(value=b_right@a.value)
             np.testing.assert_almost_equal(res.value, res2.value)
 
+    @pytest.mark.parametrize('func', [
+        operator.add,
+        operator.sub,
+        operator.mul,
+        operator.xor,  # outer product
+        operator.or_,  # inner product
+    ])
+    def test_swapped_operands(self, algebra, rng, func):    # noqa: F811
+        layout = algebra
+        for _ in range(10):
+            mv = layout.randomMV(rng=rng)
+            mv2 = layout.randomMV(rng=rng)
```

Comment:
Is that ok: https://github.com/trundev/clifford/commit/b2418f0b98e2d2633548beca693eba474d631116

> I did not push this in `fix-pygae-426` branch to avoid history pollution (if something is not ok, it can be simply amended).

BTW, there is still no test coverage for the second `return` inside `__ror__` (similar in `__or__`): https://github.com/trundev/clifford/blob/dcaa727/clifford/_multivector.py#L233
I'm not sure what cases it is intended to handle. Scalar multiplication is handled as if it is a multi-vector (converted by `_checkOther`).